data/campaigns/Sceptre_of_Fire/scenarios/8_The_Dragon.cfg

```diff
@@ -249,6 +296,10 @@
                 [/message]
             [/then]
         [/if]
+
+        # Force objectives to update
+        [show_objectives]
+        [/show_objectives]
```

Comment:
I think this is not needed.

Reply:
Apparently this is a work-around for a bug (see below) concerning when the [obectives] block processes and when it displays.

If so, this change should be disapproved and an Issue created for the problem (if we don't have one).

Reply:
Assuming that the wiki is wrong (so this is a bug), i could simply add a FIXME note so that anyone will be able to remove these lines once the bug will be fixed. Otherwise, i could simply move the [objectives] tag in the start event, and no problem then.
I find these things simpler than undoing the last commit, fixing the bug, redoing the changes.
Assuming that the wiki is right, this wouldn't be a bug. It would be quite an expected behaviour.

Personally, i'd prefer just adding a #FIXME and telling someone else to remove those lines once they are not necessary anymore.

Reply:
The wiki says displayed before the start of the first turn. That is after the prestart as well as after the start event. Note that we depend upon the objectives displaying after the start event dialog. 

The wiki is correct. (although it could be more clearly worded, as in: before or after the turn 1 begin event).

The bug is that the objectives show-if logic is performed when the tag appears (prestart, usually) and not when the popup appears.  The easy fix would be to find where the popup is forced and have it run the normal [show_objectives] process which includes testing the show-if logic.

It would be better to not put in a comment, and not work around the issue. Instead, assume it works correctly, here. and open a separate issue to get the correction in.  The subject matter of this PR is cleaning up a specific campaign, not fixing a general bug.

If you MUST work around the issue, here, then do it by moving the entire [objectives] block to appear AFTER the start event places the critical unit on the map.

Reply:
> It would be better to not put in a comment, and not work around the issue. Instead, assume it works correctly, here. and open a separate issue to get the correction in. The subject matter of this PR is cleaning up a specific campaign, not fixing a general bug.

I'll do as you say, but this bug must be solved before 1.14.5, otherwise the user will see wrong objectives.